src/VirtoCommerce.ExperienceApiModule.Core/Index/IFilterExtensions.cs

```diff
@@ -37,12 +38,17 @@ public static object MapTo(this TermFilter termFilter, object obj)
             return obj;
         }
 
-        public static string Stringify(this IFilter filter)
+        public static string Stringify(this IFilter filter, bool skipCurrency = false)
```

Comment:
Create a new method with duplicated code, instead of changing the signature of this method.